index.html

```diff
@@ -15,12 +15,72 @@
     <link rel="stylesheet" href="css/layout.css" />
     <link rel="stylesheet" href="css/styles.css" />
   </head>
+
   <body>
-    <header>
-      <div class="container"></div>
+    <header id="purple-background">
+      <div class="container">
+        <div class="row">
+          <h1 id="header-style">Code Journal</h1>
+        </div>
+      </div>
     </header>
     <main class="container">
-      <div data-view="entry-form"></div>
+      <div data-view="entry-form">
+        <form id="note-form">
+          <div class="row">
+            <div class="column-full">
+              <h2>New Entry</h2>
+            </div>
+          </div>
+          <div class="row">
+            <div class="column-half">
+              <div class="photo">
+                <label for="photo" id="img-button"></label>
```

Comment:
Since there's no text inside of here, we don't need the `label`. We only need to include `label`s if there's text associated with an `input`. Like how we have a "Photo URL" above the `input`